tests/test_interactive.py

```diff
@@ -180,12 +181,8 @@ def assert_match(result, expected, ignore_prompt_number=False):
 
     if _IPYTHON_VERSION < (1, 0):
         assert False, "we don't support IPython pre  1.0 anymore"
-    if _IPYTHON_VERSION < (0, 11):
-        assert False, "we don't support IPython pre  1.0 anymore"
     if _IPYTHON_VERSION < (0, 12):
         assert False, "we don't support IPython pre  1.0 anymore"
```

Comment:
I guess this can be removed as well, since `(0, 12) < (1, 0)` and you get the same message in either case.